libethashseal/genesis/olympic.h

```diff
@@ -0,0 +1,79 @@
+/*
+		This file is part of cpp-ethereum.
+
+		cpp-ethereum is free software: you can redistribute it and/or modify
+		it under the terms of the GNU General Public License as published by
+		the Free Software Foundation, either version 3 of the License, or
+		(at your option) any later version.
+
+		cpp-ethereum is distributed in the hope that it will be useful,
+		but WITHOUT ANY WARRANTY; without even the implied warranty of
+		MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+		GNU General Public License for more details.
+
+		You should have received a copy of the GNU General Public License
+		along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file olympic.cpp
+ * @author Gav Wood <i@gavwood.com>
+ * @date 2014
+ */
+
+#pragma once
+#include "../GenesisInfo.h"
+using namespace dev;
+using namespace eth;
+
+namespace dev
+{
+namespace eth
+{
+static h256 const c_genesisStateRootOlympic;
+static std::string const c_genesisInfoOlympic =
+R"ETHEREUM(
```

Comment:
pragma once wont help? 
how else could I split this variables into different files?